.github/workflows/ci.yaml

```diff
@@ -211,19 +216,54 @@ jobs:
         #   CARGO_TARGET_ARMV7_UNKNOWN_LINUX_GNUEABIHF_LINKER: ${{ matrix.cc }}
         #   CARGO_TARGET_X86_64_UNKNOWN_LINUX_MUSL_LINKER: ${{ matrix.cc }}
 
-      - name: Compress artifact (Not Windows)
-        if: matrix.os != 'windows-latest'
-        run: zip --junk-paths ${{ matrix.platform }} ./target/${{ matrix.target }}/release/clarinet
+      - name: Build (Windows)
+        if: matrix.os == 'windows-latest'
+        run: |
+          cargo wix -v
+          ls target/wix
 
-      - name: Compress artifact (Windows)
+      - name: Import cert to store (Windows)
         if: matrix.os == 'windows-latest'
-        run: Compress-Archive -Path ./target/${{ matrix.target }}/release/clarinet.exe -DestinationPath ${{ matrix.platform }}
+        run: |
+          $b64_cert = '${{ secrets.WINDOWS_CODE_SIGNING_CERTIFICATE }}'
+          $certificate_file_name = "${env:TEMP}\certificate.pfx"
+
+          $bytes_cert = [Convert]::FromBase64String(${b64_cert})
+          [IO.File]::WriteAllBytes(${certificate_file_name}, ${bytes_cert})
+          certutil -f -p ${{ secrets.WINDOWS_CODE_SIGNING_PASSWORD }} -importpfx ${certificate_file_name}
+
+      - name: Code Sign (Windows)
+        if: matrix.os == 'windows-latest'
+        run: |
+          $signtool_path = ((Resolve-Path -Path "${env:ProgramFiles(x86)}/Windows Kits/10/bin/10*/x86").Path)[0]
```

Comment:
![tenor (1)](https://user-images.githubusercontent.com/1447546/125844764-b07438b7-17bb-490a-893c-c0692a29388a.gif)

Reply:
Did you recall encountering [this error when trying to sign](https://github.com/hirosystems/clarinet/runs/3078571364?check_suite_focus=true#step:11:4)? I seem to have an abundance of different ways to solve this, but in each method I end up getting this error. I know the cert is valid because we use it for signing the Stacks Wallet.

Reply:
I don't recall, have you got it to work locally?

Reply:
I don't have a Windows pc configured for development at the moment, but testing in GH is slow and starting to be too much friction. I might set up my PC to debug this tonight.